packages/@aws-cdk/aws-neptune/lib/cluster.ts

```diff
@@ -360,15 +369,15 @@ export abstract class DatabaseClusterBase extends Resource implements IDatabaseC
 
   protected abstract enableIamAuthentication?: boolean;
 
-  public grantConnect(grantee: iam.IGrantable): iam.Grant {
+  public grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant {
     if (this.enableIamAuthentication === false) {
-      throw new Error('Cannot grant connect when IAM authentication is disabled');
+      throw new Error('Cannot grant when IAM authentication is disabled');
```

Comment:
I commented this on the other PR, but since I also commented that I think the wording change should happen here, adding the comment here as well. I'm concerned that this error message won't be super clear to the customer.